### PR TITLE
Cliff and vest should be able to be 0

### PR DESCRIPTION
**Pull request by wantedsystem**

### CODE REVIEW

File: src/newLaunch/seed/stages/stage4.ts [hunk 1]
```diff
@@ -158,9 +158,9 @@ export class Stage4 extends BaseStage<ISeedConfig> {
       (this.numberService.fromString(fromWei(this.launchConfig.tokenDetails.maxSupply, this.launchConfig.tokenDetails.projectTokenInfo.decimals)) *
         this.launchConfig.launchDetails.pricePerToken)) {
       message = "Funding Maximum cannot be greater than Maximum Project Token Supply times the Project Token Exchange Ratio";
-    } else if (!this.launchConfig.launchDetails.vestingPeriod || !(this.launchConfig.launchDetails.vestingPeriod >= 0)) {
+    } else if (this.launchConfig.launchDetails.vestingPeriod===null || this.launchConfig.launchDetails.vestingPeriod===undefined) {
       message = "Please enter a number greater than zero for  \"Project tokens vested for\" ";
```
Comment (dkent600) [inline src/newLaunch/seed/stages/stage4.ts:162]: message should be "greater than or equal"

File: src/newLaunch/seed/stages/stage4.ts [hunk 2]
```diff
@@ -158,9 +158,9 @@ export class Stage4 extends BaseStage<ISeedConfig> {
       (this.numberService.fromString(fromWei(this.launchConfig.tokenDetails.maxSupply, this.launchConfig.tokenDetails.projectTokenInfo.decimals)) *
         this.launchConfig.launchDetails.pricePerToken)) {
       message = "Funding Maximum cannot be greater than Maximum Project Token Supply times the Project Token Exchange Ratio";
-    } else if (!this.launchConfig.launchDetails.vestingPeriod || !(this.launchConfig.launchDetails.vestingPeriod >= 0)) {
+    } else if (this.launchConfig.launchDetails.vestingPeriod===null || this.launchConfig.launchDetails.vestingPeriod===undefined) {
```
Comment (dkent600) [inline src/newLaunch/seed/stages/stage4.ts:161]: Is there any chance can be undefined?  Maybe easier to check if is a number?  Like `Number.isInteger` and >= 0?  (note that you are allowed < 0)

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Review by dkent600 — APPROVED** [review]

**Review by dkent600 — APPROVED** [review]
@wantedsystem  Can you please fix the merge conflicts?

**Comment by wantedsystem** [comment]
@dkent600 merge conflict fixed

**Review by dkent600 — APPROVED** [review]

**Pull request merged**